baseplate/__init__.py

```diff
@@ -93,8 +96,90 @@ def make_tracing_client(raw_config, log_if_unconfigured=True):
     )
 
 
+def crash_reporter_from_config(raw_config, module_name):
```

Comment:
I think it's okay to keep as is, I can't think of anything in particular that makes this significantly cleaner.